tests/integration/test_schema.py

```diff
@@ -26,7 +27,11 @@ def test_unnoised_id_cols(dataset_name: str, request):
     unnoised_id_cols = [COLUMNS.simulant_id.name]
     if dataset_name != DATASETS.ssa.name:
         unnoised_id_cols.append(COLUMNS.household_id.name)
-    data = request.getfixturevalue(f"sample_data_{dataset_name}")
+    if dataset_name == DatasetNames.TAXES_1040:
```

Comment:
Can you move this logic into the `_load_sample_data` function so that you always just call that? And then if the dataset is 1040, then that function knows to call the formatting stuff?